sage/pallet/src/tests/extrinsics/buy_asset.rs

```diff
@@ -56,7 +56,7 @@ fn buy_should_work() {
 				asset_for_sale,
 				asset_price
 			));
-			assert_ok!(Sage::buy_asset(RuntimeOrigin::signed(ALICE), asset_for_sale));
+			assert_ok!(Sage::buy_asset(RuntimeOrigin::signed(ALICE), asset_for_sale, Some(NATIVE)));
```

Comment:
`Some(Native)` is actually the same as `None`, but I wanted it to be explicit for better readability.